.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+Please include the following information when opening an issue:
```

Comment:
Agree that it shouldn't bee too rigid! For me, the often missed contents are:

1. Branch or installer method
1. Traceback or relevant snippet from server.log
1. How to reproduce
1. Screenshots
1. Real-life consequences (anything community should be aware of about how it affects your deployment)